.github/workflows/ci.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
             matrix:
                 folder:
                     - '.'
-                    - 'e2e/bzlmod'
+                    # - 'e2e/bzlmod' # FIXME
```

Comment:
the fact that we can't fix it now is an indicator that the changes were naughty to make during rc :)

Reply:
The error message was for `js/libs.bzl.
```
ERROR: error loading package '': at /private/var/tmp/_bazel_greg/77e9eabe4dbb96758b221045a8e291a6/external/aspect_rules_ts/ts/defs.bzl:14:6: at /private/var/tmp/_bazel_greg/77e9eabe4dbb96758b221045a8e291a6/external/aspect_rules_ts/ts/private/ts_project.bzl:6:6: cannot load '@aspect_rules_js.0.13.0//js:libs.bzl': no such file
```
Sad we can't add a new entry in bzlmod because of the rc suffix

Reply:
Fixed with 0.14.0 in BCR on same commit that will be rc4